mcs/class/System/Test/System.Diagnostics/PerformanceCounterTests.cs

```diff
@@ -0,0 +1,124 @@
+// originally copied from external/corefx
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Specialized;
+using NUnit.Framework;
+using System.Diagnostics;
+
+namespace MonoTests.System.Diagnostics.TestPerformanceCounter
```

Comment:
should be MonoTests.System.Diagnostics.PerformanceCounterTests to follow the conventions we have in most other places.